views/default/input/date.js

```diff
@@ -0,0 +1,62 @@
+define(function (require) {
+
+	var elgg = require('elgg');
+	var $ = require('jquery');
+	require('jquery-ui');
+
+	if (elgg.get_language() !== 'en') {
+		// require language module if necessary
+		require('jquery-ui/i18n/datepicker-' + elgg.get_language() + '.min');
```

Comment:
Within a define function RequireJS uses `Function.toString()` trickery to allow synchronous-looking code like `require('foo')`, but it's not really synchronous. The `'foo'` is discovered and loaded before the define function is called.